### ISSUE TITLE
Add option to hide EDDN data / distinguish data somehow

### ISSUE THREAD

**Issue by Lknechtli** [stringandstickytape/RegulatedNoise#29]
Generally if you've been to a system you can count on the data you import being somewhat correct, but the same can't be said for data from EDDN.
There should be some way to hide / ignore said data, or even clear it separately from data you have gathered yourself.

**Comment by sutex** [comment]
From Suggestions ,in Issues 

" Bad data ie 001Tan can be deleted along with the station IF! it's an EDDN import as that station is not in my Commander Log ( I may have flown in an out of that system) otherwise once clicked takes me to THAT station so I can edit that commodity or delete the associated station. Maybe the screenshots are deleted only after leaving
By Stations &/or By Commodities tab, RN then assumes you have check , if highlight didn't show any errors" 

Also , If DEV's says no EDDN Crowd sourcing allowed , & they might , It is important now to have EDDN separated  from users collected data , while your still building it 

Other is to have list of Users names that you will allowed EDDN imports from ie friends

**Comment by stringandstickytape** [comment]
There's already a "purge EDDN data" button on the EDDN tab, but it wouldn't be hard to add a checkbox for "include EDDN data" in the "Stations within x light years of y" set at the top of the Price Analysis tab.  Much of the plumbing already in there.

"DEV's says no EDDN Crowd sourcing allowed , & they might" - agree, we need to be mindful of this.  EDDN data is now identified as such in the Filename field ("<From EDDN>") so we have that data should we need it in the future.

Whitelist (and, I daresay, blacklist) of usernames is a nice idea, I will split that into a separate issue.
